tasks/kyu7.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+"""Python CodeWars task. 7-kyu
+        https://www.codewars.com/kata/sum-of-angles
+"""
+
+
+def angle(_num_of_sides):
+    """find the total sum of angles in an n sided shape.
+        Args:
+            _num_of_sides (int): number of sides.
+
+        Returns:
+            int: Total sum of angles.
+    """
+    return 180 * (_num_of_sides - 2)
```

Comment:
i think 180 it's a const, please declare this before function(uppercase)

Reply:
Why declaring a variable outside of the function  if we only need it inside the function?

Reply:
> Why declaring a variable outside of the function if we only need it inside the function?

Because, it's a const, and it does not change. 
It's my opinion, if you have some const, you should to name it, so that another programmer can understand what is it.

Reply:
I meant that there is no need in declaring it outside of the function, but it still can be declared inside of the function

Reply:
Ok, I`ve declared